app/controllers/games_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def create
     game = Game.create(white_player_id: current_user.id)
+    current_user << game
```

Comment:
^ Nevermind, if AR methods didn't work in controllers, your second example wouldn't work either.